modules/utxo-lib/test/address/ecash/ecashaddr.ts

```diff
@@ -0,0 +1,17 @@
+import { fromOutputScriptWithFormat, toOutputScriptWithFormat } from '../../../src/addressFormat';
+import { networks } from '../../../src/networks';
+import * as assert from 'assert';
+
+// Addresses used from https://www.npmjs.com/package/ecashaddrjs
+const ecashAddress = 'ecash:qpadrekpz6gjd8w0zfedmtqyld0r2j4qmuthccqd8d',
+  bitcoinCashAddress = 'bitcoincash:qpadrekpz6gjd8w0zfedmtqyld0r2j4qmuj6vnmhp6',
+  outputScriptHex = '76a9147ad1e6c11691269dcf1272ddac04fb5e354aa0df88ac';
+
+describe('ecash address', () => {
```

Comment:
nit: https://mochajs.org/#arrow-functions

Reply:
Done

Reply:
Ah I guess we don't have the mocha lint rule in this repo (yet?).